src/components/BulkEditList/BulkEditListResult/BulkEditInApp/forms/HoldingsForm/HoldingsForm.js

```diff
@@ -121,66 +121,70 @@ export const HoldingsForm = (
       fields={fields}
       className={css.row}
       onAdd={noop}
-      renderField={(field, index) => (
-        <Row data-testid={`row-${index}`}>
-          <Col xs={3} sm={3}>
-            <Select
-              dataOptions={field.options}
-              value={field.option}
-              onChange={(e) => handleSelectChange(e, index, fieldsTypes.OPTION)}
-              data-testid={`select-option-${index}`}
-            />
-          </Col>
-          <Col xs={2} sm={2}>
-            <Select
-              dataOptions={field.actions}
-              value={field.action}
-              onChange={(e) => handleSelectChange(e, index, fieldsTypes.ACTION)}
-              data-testid={`select-actions-${index}`}
-              disabled={isDisabled(index)}
-            />
-          </Col>
-          {isLocation(index) &&
-            <Col xs={6} sm={3}>
-              <LocationSelection
-                value={field.locationId}
-                onSelect={(location) => handleLocationChange(location, index)}
-                placeholder={intl.formatMessage({ id: 'ui-bulk-edit.layer.selectLocation' })}
-                data-test-id={`textField-${index}`}
+      renderField={(field, index) => {
+        const isAddButtonShown = index === fields.length - 1 && fields.length !== optionsHoldings.length - 1;
```

Comment:
mb it's better to move it to utils, to not copypaste to new forms

Reply:
Thanks, refactored